libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -79,11 +79,12 @@
     #endif
 
     #define DECLARE_FOR_ALL_TYPES(foo)\
-	   foo(char) json_nothrow;	 foo(unsigned char) json_nothrow;\
-	   foo(short) json_nothrow;	 foo(unsigned short) json_nothrow;\
-	   foo(int) json_nothrow;	 foo(unsigned int) json_nothrow;\
-	   foo(long) json_nothrow;	 foo(unsigned long) json_nothrow;\
-	   foo(float) json_nothrow;	 foo(double) json_nothrow;\
+       foo(char) json_nothrow;      foo(unsigned char) json_nothrow;\
```

Comment:
I see, the old code contains tabs, you added spaces. 
I would argue that we should use spaces... but we can take this into a separate discussion. 
I would propose to use clang_format in order to have a common format for all of us, otherwise we will diverge very soon. But I guess we can ignore the issue for this PR